integration_test/third_party_apps_data/applications/elasticsearch/metadata.yaml

```diff
@@ -102,3 +103,43 @@
   labels:
     state: .*
     thread_pool_name: .*
+expected_logs:
+- log_name: elasticsearch_json
+  fields:
+  - name: jsonPayload.message
+    value_regex: initialized
+    type: string
+    description: 'Log message'
+  - name: jsonPayload.component
+    value_regex: o.e.n.Node
```

Comment:
These are intended to be literal `.` characters, so they need to be escaped.

Reply:
(Since it was like that before, I'd be OK with doing a follow-up PR that does a deeper pass over the YAML files to resolve these kinds of issues. We don't need to hold up this PR for things that were already there.)